Tests iOS/Support/Elements/PocketUIElement.swift

```diff
@@ -27,7 +27,7 @@ extension PocketUIElement {
 
     @discardableResult
     func wait(
-        timeout: TimeInterval = 2,
+        timeout: TimeInterval = 30,
```

Comment:
While aggressive, our tests should never really wait this long, HOWEVER, i found that there are some cases where Sails (the web faker we use), locks up and takes a few to respond halting the app loading.

Reply:
Do we have actual evidence that increasing the wait that much will make some tests pass when they normally don't? And if it's a Sails problem, do we want to use a timeout that long?
The reason I'm asking is because, if tests fail, then they'll be waiting 30 seconds before actually exit, and because this call is common to a lot of tests, this might increase test suites execution times by a significant amount. 
My 2 cents is that a reasonable time interval represents a "healthy" test. If we need to tweak the timeout to 30 seconds, it feels like something is wrong somewhere else.
How about we keep 2 seconds and monitor the situation? Your changes made a lot of improvements so we might be able to better pinpoint latency issues, what do you think?